Dockerfile.kernel

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) Jupyter Development Team.
 # Distributed under the terms of the Modified BSD License.
 
-FROM jupyter/pyspark-notebook:0017b56d93c9
+FROM jupyter/all-spark-notebook:0017b56d93c9
 
 RUN pip install jupyter_kernel_gateway==0.5.0
 # install ipywidgets 5.x
```

Comment:
This is where work needs to be done:

1. Declarative widgets and ipywidgets need to be installed against python2 environment.
2. Declarative widgets JAR needs to be installed against the Toree kernel.